examples/3d/z_sort_debug.rs

```diff
@@ -70,7 +70,7 @@ fn setup(
                         shaded: false,
                         ..Default::default()
                     }),
-                    translation: Translation::new(0.0, 3.0, 0.0),
+                    transform: Translation::new(0.0, 3.0, 0.0).into(),
```

Comment:
You should probably get rid of the Translation, Rotation, and Scale types, and just add from_translation, from_rotation, and from_scale to Transform.

Reply:
Good point. So you mean e.g. represent a translation just by a vec3 and a rotation just by a quat?